code/game/machinery/camera/camera_assembly.dm

```diff
@@ -87,6 +87,9 @@
 	return TRUE
 
 /obj/structure/camera_assembly/screwdriver_act(mob/user, obj/item/tool)
+	. = ..()
+	if(.)

```

Comment:
You can clean all these up just with `if((. = ..())`, I believe.

Reply:
I would do that but I'm worried the syntax would confuse someone so I'm going to hold off on that unless I'm asked to by a maintainer.

Reply:
Yeah don't assign vars in conditionals